.travis.yml

```diff
@@ -5,7 +5,8 @@ node_js:
 - '4'
 before_install:
 - npm install -g npm
-script: npm test
+script: COVERAGE=y npm test
+after_script: "npm install coveralls@2.11.8 && cat ./coverage/lcov.info | coveralls"
```

Comment:
@kumar303 yep, we can, but as far I read from the coveralls docs (which I'm reading right now for the first time and so it's perfectly possible that I'm wrong) it needs a separate config file with the developer account keys to work locally, so I thought to reduce the amount of dependencies installed locally but installing it directly on travis
(but I'm not against to add it into the devDependencies)